python/desc/sims/GCRCatSimInterface/DatabaseEmulator.py

```diff
@@ -381,8 +377,12 @@ def _transform_knots(self, gc):
 
         # Apply flux correction for the random walk
         add_postfix = []
+        
         for name in gc.list_all_native_quantities():
```

Comment:
Line 381: I think this needs to be changed to:
```python
    for name in gc.list_all_quantities(include_native=True):
```
so that it returns both native and derived quantities.